swap/cheque.go

```diff
@@ -83,7 +83,7 @@ func (cheque *Cheque) VerifySig(expectedSigner common.Address) error {
 	return nil
 }
 
-// Sign signs the cheque with supplied private key
+// Sign returns a signature for the cheque with supplied private key
```

Comment:
Isn't it more correct to say something that it returns the signed cheque? Not really "just" a signature

Reply:
where is the cheque returned, though?

Reply:
updated signature comment.